internal/js/modules/k6/browser/common/network_manager.go

```diff
@@ -815,13 +821,12 @@ func (m *NetworkManager) Authenticate(credentials Credentials) error {
 	return nil
 }
 
-func (m *NetworkManager) AbortRequest(requestID fetch.RequestID, errorReason string) {
+func (m *NetworkManager) AbortRequest(requestID fetch.RequestID, errorReason string) error {
```

Comment:
I'm not sure whether we should return an error from these functions. There are many requests, and failing one would kill the whole test run. It's good to fail fast, but in practice, it might not be best for user experience. That said, we can always revert this change if it becomes a problem later. I believe it's fine for now.